src/Test/BaseIntegrationTestCase.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+namespace Twig\Test;
+
+use PHPUnit\Framework\TestCase;
+use Twig\Environment;
+use Twig\Error\Error;
+use Twig\Extension\ExtensionInterface;
+use Twig\Loader\ArrayLoader;
+use Twig\RuntimeLoader\RuntimeLoaderInterface;
+use Twig\TwigFilter;
+use Twig\TwigFunction;
+use Twig\TwigTest;
+
+abstract class BaseIntegrationTestCase extends TestCase
```

Comment:
The base class should be tagged as `@internal` to allow us to remove it again in the future when we don't need it anymore.